packages/client/src/index.ts

```diff
@@ -28,9 +27,10 @@ export function getSignature(
 ): DecryptedMessage | null {
   const key = getGeneratedRawKey(storage, password);
   const decryptedMessage = getDecryptedMessage(key, storage);
-  if (decryptedMessage === null) return null;
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-  return JSON.parse(decryptedMessage);
+  return decryptedMessage.matches({
+    present: (message) => JSON.parse(message) as DecryptedMessage,
+    empty: () => null
+  });
```

Comment:
When there is a value, do JSON.parse and return, and when there is no value, it returns Null.